docs/sources/whatsnew/whats-new-in-v8-0.md

```diff
@@ -78,6 +80,23 @@ Log navigation in Explore has been significantly improved. We added pagination t
 
 You can now use the Plugin catalog app to easily manage your plugins from within Grafana. Install, update, and uninstall plugins without requiring a server restart.
 
+### Performance improvements
+
+Grafana 8.0 includes many performance enhancements.
+
+#### Initial startup and load performance
+
+We reduced the Grafana initial download size massively, approximately 40%. This means that on slower or mobile connections, the initial login page or home dashboard will load much faster. 
+
+All panels that have migrated from Flot to uPlot will also render two to three times faster because the library is much more efficient. Right now, this includes the Time series, Stat, Timeline, Histogram, and Barchart panel visualizations.
```

Comment:
we can probably drop `because the library is much more efficient`, seems redundant.

not sure if it's relevant, but some of these panels are completely new (Bar chart, Timeline), so they didn't really migrate from Flot. we're also breaking up single Timeline panel into two panels today: `State timeline` and `Status grid`: https://github.com/grafana/grafana/pull/34171. sorry for all the churn. naming is hard!

Reply:
Well, since nobody has shown me the Timeline panel yet or given me any docs or info, there's nothing for me to update. (hint hint)